modules/admin_manual/pages/installation/manual_installation/manual_installation_apache.adoc

```diff
@@ -35,102 +35,87 @@ Alias /owncloud "/var/www/owncloud/"
 Then create a symlink to /etc/apache2/sites-enabled:
 
 ----
-ln -s /etc/apache2/sites-available/owncloud.conf /etc/apache2/sites-enabled/owncloud.conf
+sudo ln -s /etc/apache2/sites-available/owncloud.conf /etc/apache2/sites-enabled/owncloud.conf
 ----
 
 == Additional Apache Configurations
 
-* For ownCloud to work correctly, we need the module `mod_rewrite`.
-Enable it by running: `a2enmod rewrite`. Additionally recommended modules are
-`mod_headers`, `mod_env`, `mod_dir`, `mod_mime`, and `mod_unique_id`.
-To enable them, run the following commands:
+* For ownCloud to work correctly, we need the module `mod_rewrite`. Enable it by running: `a2enmod rewrite`. Additionally recommended modules are `mod_headers`, `mod_env`, `mod_dir`, `mod_mime`, and `mod_unique_id`. To enable them, run the following commands:
 +
-....
-  a2enmod headers
-  a2enmod env
-  a2enmod dir
-  a2enmod mime
-  a2enmod unique_id
-....
+[source,console]
+----
+sudo a2enmod headers
+sudo a2enmod env
+sudo a2enmod dir
+sudo a2enmod mime
+sudo a2enmod unique_id
+----
 +
-NOTE: If you want to use the {oc-marketplace-url}/apps/oauth2[the OAuth2 app], then {mod_headers_url}[mod_headers] must be installed and enabled.
+NOTE: If you want to use the {oc-marketplace-url}/apps/oauth2[OAuth2 app], then {mod_headers_url}[mod_headers] must be installed and enabled.
 
-* You must disable any server-configured authentication for ownCloud, as it uses Basic authentication
-  internally for DAV services. If you have turned on authentication on a parent folder
-  (via, e.g., an `AuthType Basic` directive), you can disable the authentication specifically for the ownCloud entry. 
-  Following the above example configuration file, add the following line in the `Directory` section
+* You must disable any server-configured authentication for ownCloud, as it uses Basic authentication   internally for DAV services. If you have turned on authentication on a parent folder (via, e.g., an `AuthType Basic` directive), you can disable the authentication specifically for the ownCloud entry. Following the above example configuration file, add the following line in the `Directory` section above:
```

Comment:
```suggestion
* You must disable any server-configured authentication for ownCloud since it uses basic authentication internally for DAV services. If you have turned on authentication on a parent folder (e.g. via an `AuthType Basic` directive), you can disable the authentication specifically for the ownCloud entry. As in the above example configuration file, add the following line in the `Directory` section:
```